### PR TITLE
Create bug bounty program page in Lisk docs - Closes #1506 

### LINKED ISSUE #1506: Add page about the bug bounty program
### Is your feature request related to a problem?
The bug bounty page from lisk.com will be removed, so we need to add it to the Lisk docs.

### Description
A page explaining the Lisk bounty program, analog to the currently existing page https://lisk.com/bug-bounty-program

### PR DESCRIPTION
**Pull request by chris529**
Closes #1506

### CODE REVIEW

File: docs/modules/ROOT/pages/intro/bug-bounty-program.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+= Bug bounty program
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/intro/bug-bounty-program.adoc]:
```suggestion
= Lisk Bug Bounty Program
```

File: docs/modules/ROOT/pages/intro/bug-bounty-program.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Chris Braithwaite <christopher.braithwaite@lightcurve.io> Mona Bärenfänger <mona@lightcurve.io>
+:description: This page covers the submission guidelines regarding how to receive remuneration for locating and reporting security vulnerabilities and bugs in the Lisk core.
+:toc: preamble
+:idprefix:
+:idseparator: -
+:imagesdir: ../../assets/images
+
+:url_submission_form: https://lisk.com/contact/ive-found-bug/form
+
+image:intro/bug-bounty-update.jpeg[align="center"]
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/intro/bug-bounty-program.adoc]:
```suggestion
image::intro/bug-bounty-update.jpeg[align="center"]
```

use block image syntax here

File: docs/modules/ROOT/pages/intro/bug-bounty-program.adoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+=== Validity of vulnerabilities
+
+Vulnerabilities that were already submitted, are already known to us, or are fixed by the implementation of an existing LIP are not eligible for any remuneration.
+Serial vulnerabilities caused by the same underlying issue are treated as a single vulnerability.
+
+=== Remuneration
+
+The following table illustrated below lists the current remuneration structure for identifying bugs and vulnerabilities.
+
+image:intro/remuneration1.png[ align="center"]
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/intro/bug-bounty-program.adoc]:
```suggestion
image::intro/remuneration1.png[ align="center"]
```

File: docs/modules/ROOT/pages/intro/bug-bounty-program.adoc [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+=== Report your Bug
+
+To file a report, use this link to access the {url_submission_form}[submission form^].
+Alternatively, if you prefer to file a report via email, use the following address: security@lisk.com
+
+image:intro/bug-report-form.png[align="center" ,400]
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/intro/bug-bounty-program.adoc]:
```suggestion
image:intro/bug-report-form.png[role="related thumb right",200]
To file a report, use this link to access the {url_submission_form}[submission form^].
Alternatively, if you prefer to file a report via email, use the following address: security@lisk.com
```
to improve image/text alignment

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please fix the existing conflicts in `nav.adoc`, and see my comments below

File: docs/modules/ROOT/pages/intro/bug-bounty-program.adoc [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+:idseparator: -
+:imagesdir: ../../assets/images
+
+:url_submission_form: https://lisk.com/contact/ive-found-bug/form
+
+image::intro/bug-bounty-update.jpeg[align="center"]
+
+== Overview
+
+The Lisk bug bounty program is a long-standing initiative designed to offer an incentive in the form of a remuneration payment to anyone who locates any bugs or vulnerabilities that may exist within the existing Lisk core network.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/intro/bug-bounty-program.adoc]:
```suggestion
The Lisk bug bounty program is a long-standing initiative designed to offer an incentive in the form of a remuneration to anyone who locates any bugs or vulnerabilities that may exist within the existing Lisk core network.
```

File: docs/modules/ROOT/pages/intro/bug-bounty-program.adoc [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+Chris Braithwaite <christopher.braithwaite@lightcurve.io> Mona Bärenfänger <mona@lightcurve.io>
+:description: This page covers the submission guidelines regarding how to receive remuneration for locating and reporting security vulnerabilities and bugs in the Lisk core.
+:toc: preamble
+:idprefix:
+:idseparator: -
+:imagesdir: ../../assets/images
+
+:url_submission_form: https://lisk.com/contact/ive-found-bug/form
+
+image::intro/bug-bounty-update.jpeg[align="center"]
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/intro/bug-bounty-program.adoc]: The quality of the images gets compromised when zoomed in. Please use the images from the following URLs:

- https://lisk.com/sites/default/files/media/bugbounty-hero_6.svg
- https://lisk.com/sites/default/files/pictures/2020-03/Bugbounty-payouttable-v2.svg
- https://lisk.com/sites/default/files/pictures/2022-06/Group%201062.png

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/intro/bug-bounty-program.adoc (hunk 5 above)
Comment (chris529) [reply docs/modules/ROOT/pages/intro/bug-bounty-program.adoc]: Thanks for the feedback. If you wish to remove the word  'payment' then we would also need to remove the indefinite article 'a' as well, as this is then redundant when used with an uncountable noun such as 'remuneration'
I  agree and have updated it accordingly 👍

File: docs/modules/ROOT/pages/intro/bug-bounty-program.adoc (hunk 6 above)
Comment (chris529) [reply docs/modules/ROOT/pages/intro/bug-bounty-program.adoc]: Cool..Many thanks for that Talha. I will replace them 👍

**Comment by Tschakki** [comment]
Just noticed, the base branch for this PR should be `main`, not `development`, because we need this page in the live docs.

For the `nav.adoc` file, this means the `Intro` section doesn't exist yet, so I would add it under `Products` in the nav.

**Comment by Tschakki** [comment]
@chris529 it seems the base branch change results in a lot of unwanted changes to be part of this PR.

Please undo all changes not relevant for your PR. The probably easiest way to do it is to open a new PR on the base of the `main` branch, and just close this PR.

**Comment by chris529** [comment]
This PR has been closed as it was originally on the development branch, and a new PR has now been created with the same content on the main branch.

**Pull request closed without merging**